src/stravalib/tests/unit/test_lat_long.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from stravalib.model import LatLon, SummaryActivity
+
+
+@pytest.mark.parametrize(
+    "start_latlng, end_latlng, start_expected_return, end_expected_return",
+    [
+        (
+            [10.2, 1.02],
+            [10.2, 1.02],
+            LatLon(root=[10.2, 1.02]),
+            LatLon(root=[10.2, 1.02]),
+        ),
+        (None, None, None, None),
+        ([], [], LatLon(root=None), LatLon(root=None)),
```

Comment:
this test was failing because i thought it woudl return none but really it returns a LatLon object with root = None